compiler/optimizer/DeadTreesElimination.cpp

```diff
@@ -629,7 +629,7 @@ void TR::DeadTreesElimination::prePerformOnBlocks()
           && tt->getPrevTreeTop()->getNode()->getOpCodeValue() == TR::BBStart
           && !tt->getPrevTreeTop()->getNode()->getBlock()->isExtensionOfPreviousBlock())
          {
-         requestOpt(OMR::redundantGotoElimination, tt->getEnclosingBlock());
+         requestOpt(OMR::redundantGotoElimination, true, tt->getEnclosingBlock());
```

Comment:
This is also a bug. We're passing in the block pointer as a boolean which determines whether we want to request this optimization, however we're requesting it on a `NULL` block instead of the block we want it in!